app/config/parameters.yml.dist

```diff
@@ -33,5 +33,7 @@ parameters:
       - from_id: from-subject
         to_id: to-subject
         to_name: To Subject Name
+    rds_articles:
+        30274: https://repro.elifesciences.org/example.html
```

Comment:
Should this key `30274` also be a string like in `parameters.yml`?

Reply:
Yes, I didn't recall modifying this as I just found out my .gitignored `parameters.yml` is a symlink to this